apps/prairielearn/src/components/Navbar.html.ts

```diff
@@ -14,11 +14,13 @@ export function Navbar({
   navPage,
   navSubPage,
   navbarType,
+  marginBottom = true,
```

Comment:
I had to rework how the bottom margins worked to avoid extraneous margins given this page's special layout.